nav2_simple_commander/launch/inspection_demo_launch.py

```diff
@@ -17,27 +17,34 @@
 from ament_index_python.packages import get_package_share_directory
 
 from launch import LaunchDescription
-from launch.actions import ExecuteProcess, IncludeLaunchDescription
+from launch.actions import DeclareLaunchArgument, IncludeLaunchDescription
 from launch.launch_description_sources import PythonLaunchDescriptionSource
 from launch_ros.actions import Node
 
 
 def generate_launch_description():
     warehouse_dir = get_package_share_directory('aws_robomaker_small_warehouse_world')
     nav2_bringup_dir = get_package_share_directory('nav2_bringup')
-    python_commander_dir = get_package_share_directory('nav2_simple_commander')
+    gazebo_ros = get_package_share_directory('gazebo_ros')
 
     map_yaml_file = os.path.join(warehouse_dir, 'maps', '005', 'map.yaml')
-    world = os.path.join(python_commander_dir, 'warehouse.world')
```

Comment:
@lucabonamini warehouse.world file has turtlebot model as well as aws world. by switching to aws pkg for the world file. turtlebot model has to be spawned seperately using gazebo spawner. we can use turtlebot model file in nav2 bringup. this goes for all the launch files in simple commander

Reply:
@sathak93 You're right. I'm already working on fix ti.

Reply:
Fixed.